File: app/src/main/java/com/uncmorfi/dialogs/AboutDialog.java
package com.uncmorfi.dialogs;


import android.app.Dialog;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.v4.app.DialogFragment;
import android.support.v7.app.AlertDialog;
import android.view.LayoutInflater;
import android.view.View;

import com.uncmorfi.R;

public class AboutDialog extends DialogFragment {

    public AboutDialog() {
    }

    @Override
    @NonNull
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        return createNewCardDialog();
    }

    public AlertDialog createNewCardDialog() {
        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());

        LayoutInflater inflater = getActivity().getLayoutInflater();

        View v = inflater.inflate(R.layout.dialog_about, null);

        builder.setView(v);

        return builder.create();
    }

}

File: app/src/main/java/com/uncmorfi/ui/about/AboutDialog.kt
package com.uncmorfi.ui.about

import android.app.Dialog
import android.os.Bundle
import android.view.LayoutInflater
import androidx.fragment.app.DialogFragment
import androidx.appcompat.app.AlertDialog
import com.uncmorfi.BuildConfig
import com.uncmorfi.databinding.DialogAboutBinding

class AboutDialog : DialogFragment() {

    private val version: String = BuildConfig.VERSION_NAME

    private lateinit var binding : DialogAboutBinding

    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        val builder = AlertDialog.Builder(requireActivity())

        binding = DialogAboutBinding.inflate(LayoutInflater.from(context),null,false)
        binding.versionName.text = version

        builder.setView(binding.root)
        return builder.create()
    }
}